sdk/deviceupdate/arm-deviceupdate/CHANGELOG.md

```diff
@@ -1,15 +1,22 @@
 # Release History
+    
+## 2.0.0 (2023-12-05)
+    
+**Features**
 
-## 1.1.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+  - Added Interface AccountsDeleteHeaders
+  - Added Interface Encryption
+  - Added Interface InstancesDeleteHeaders
+  - Added Interface PrivateEndpointConnectionProxiesDeleteHeaders
+  - Added Interface PrivateEndpointConnectionsDeleteHeaders
+  - Added Type Alias SKU
+  - Interface Account has a new optional parameter encryption
 
-### Other Changes
+**Breaking Changes**
 
+  - Type of parameter sku of interface Account is changed from Sku to SKU
```

Comment:
Not sure whether it's a codegen breaking

Reply:
the breaking doesn't make sense to me, could you use directive to avoid it?